ui/components/src/ui-components/filtering/FilteringInput.tsx

```diff
@@ -5,6 +5,7 @@
 /** @packageDocumentation
  * @module Filtering
  */
+/* eslint-disable deprecation/deprecation */
```

Comment:
This should not be used on source files. Deprecation warnings may be suppressed per-file in tests or per-line in source files.

Reply:
Changed to per-line